backend/services/database_service.py

```
@@ -102,6 +103,53 @@ def get_security_findings(self, limit: int = 100, offset: int = 0) -> list:
+                            SecurityFinding.resolved.is_(True),
+                            SecurityFinding.status.in_(["RESOLVED", "CLOSED"]),
+                        )
+                    )
+                if "SUPPRESSED" in statuses:
+                    status_conds.append(SecurityFinding.status == "SUPPRESSED")
+                if status_conds:
+                    q = q.filter(or_(*status_conds))
+            findings = q.order_by(SecurityFinding.created_at.desc()).all()
+            session.close()
+            return findings
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)